docs/analytics/quickstart.mdx

```diff
@@ -2,52 +2,76 @@
 title: Quickstart
 ---
 
-## Overview
+<Note>
+  ParadeDB Analytics is powered by our new extension, `pg_lakehouse`. This
+  extension replaces `pg_analytics`, our original extension. Documentation for
+  `pg_analytics` has been [preserved](/analytics_deprecated/quickstart) for
+  existing users.
+</Note>
 
-Regular Postgres tables, known as heap tables, organize data by row. While this makes sense for operational data,
-it is inefficient for analytical queries, which often scan a large amount of data from a subset of the columns
-in a table.
+## Overview
 
-ParadeDB introduces special tables called `parquet` tables. These tables behave like regular Postgres tables but
-have two primary advantages:
+ParadeDB Analytics transforms Postgres into a fast analytical query engine over external object stores like Amazon S3, file
+formats like Parquet, and table formats like Delta Lake. Queries are pushed down to [Apache DataFusion](https://github.com/apache/datafusion), which
```

Comment:
Same comment here, perhaps we can mention local file system is how people who want to do what `pg_analytics` used to be for can use?

Reply:
Hmm I'm not sure if that's the case. In my mind the local file system is only useful for quick testing.